apps/molgenis-components/src/components/tables/AggregateTable.vue

```diff
@@ -191,9 +192,9 @@ export default defineComponent({
       if (this.refColumns?.length > 0) {
         this.selectedColumn = this.refColumns[0];
         this.selectedRow = this.refColumns[1] || this.refColumns[0];
+        //this.fetchData();
```

Comment:
remove

Reply:
this is the place where my issue with massive reloading exists and where I might need help